ShapeCrawler/PowerPoint/ISlide.cs

```diff
@@ -41,6 +43,11 @@ public interface ISlide
         
         IShapeCollection Shapes { get; }
 
+        /// <summary>
+        /// Gets a list of all textboxes on that slide, including those in tables
+        /// </summary>
+        public IList<ITextBox> Textboxes { get; }
```

Comment:
@femo1de, thank you for the PR.

I think the _method_ concept, eg. `GetAllTextboxes()` or `GetTextBoxList()`, would be more suitable here. Textboxes is not _direct property_ of the slide from user's perspective.

Reply:
@femo1de, did you have chance to resolve the review comments?

Reply:
@adamshakhabov they should all be solved now

Reply:
Pull Request contains a lot of other changes:

![image](https://user-images.githubusercontent.com/5969304/191989858-629d78fa-854c-4c6c-b5a8-5515928836bd.png)

Looks like the conflict resolving while `merge`/`rebase` operation was made incorrectly. For example, the "Files changed" tab says, the following changes are your, which is not true:

![image](https://user-images.githubusercontent.com/5969304/191990608-e4e1b9f8-7df6-4027-b1ac-672d001b746c.png)